subsys/bluetooth/le_audio/audio_encoder.c

```diff
@@ -71,16 +83,15 @@ static void audio_encoder_thread_func(void *p1, void *p2, void *p3)
 		/* A NULL audio block can be sent to the queue to wake up and abort thread. Continue
 		 * and check thread abort flag.
 		 */
-		if (audio == NULL) {
-			continue;
+		if (unlikely(audio == NULL)) {
```

Comment:
to follow your other changes:
```suggestion
		if (unlikely(!audio)) {
```

Reply:
Done.